projects/cn0540/de10nano/system_project.tcl

```diff
@@ -0,0 +1,69 @@
+
+source ../../scripts/adi_env.tcl
+source ../../scripts/adi_project_intel.tcl
+
+adi_project cn0540_de10nano
+
+source $ad_hdl_dir/projects/common/de10nano/de10nano_system_assign.tcl
+
+# files
+
+# SPI interface for ad7768-1
+
+set_location_assignment PIN_AH12 -to cn0540_spi_sclk      ; ##   Arduino_IO13
+set_location_assignment PIN_AH11 -to cn0540_spi_miso      ; ##   Arduino_IO12
+set_location_assignment PIN_AG16 -to cn0540_spi_mosi      ; ##   Arduino_IO11
+set_location_assignment PIN_AF15 -to cn0540_spi_cs        ; ##   Arduino_IO10
+
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to cn0540_spi_sclk
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to cn0540_spi_miso
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to cn0540_spi_mosi
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to cn0540_spi_cs
+
+# I2C
+
+set_location_assignment PIN_AG11  -to i2c_scl             ; ##   Arduino_IO15
+set_location_assignment PIN_AH9   -to i2c_sda             ; ##   Arduino_IO14
+
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to i2c_scl
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to i2c_sda
+
+# SPI interface for ltc2308
+
+set_location_assignment PIN_E8   -to dbg_ltc2308_sclk   ; ## GPIO_0[1] - pin 2
+set_location_assignment PIN_D11  -to dbg_ltc2308_miso   ; ## GPIO_0[3] - pin 4
+set_location_assignment PIN_AH13 -to dbg_ltc2308_mosi   ; ## GPIO_0[5] - pin 6
+set_location_assignment PIN_AH14 -to dbg_ltc2308_cs     ; ## GPIO_0[7] - pin 8
+
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to dbg_ltc2308_sclk
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to dbg_ltc2308_miso
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to dbg_ltc2308_mosi
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to dbg_ltc2308_cs
+
```

Comment:
These pins I think can be removed

Reply:
Those pins will be removed at the point when the interface is fully functional.

Reply:
My bad, it was a left over, thanks!